src/api/db/schemas/Project.js

```diff
@@ -60,6 +61,13 @@ const DeploymentSchema = new Schema({
   editable: { type: Boolean }
 });
 
+const ProjectLabelSchema = new Schema({
+  _id: { type: String, required: true, default: randomUUID },
```

Comment:
I don't think we need to use `randomUUID` unless you don't like MongoDB's `ObjectIds` for some reason. You don't even have to define the `_id` field in the schema, Mongoose will add it for you if you omit it: https://mongoosejs.com/docs/guide.html#_id

Reply:
I think the only reason to not use `ObjectIds` would be if you want to set it to something that has meaning for other reasons, like it needs to be human readable or it's a camera serial number or something.